src/components/filter/row/MultipleStringFIlterRow.tsx

```diff
@@ -153,7 +156,9 @@ export default function MultipleStringFilterRow({ config }: MultipleStringFilter
 						value={config.currentValue}
 						setValue={config.setCurrentValue}
 						autoresize
-						autoCompleteList={config.autocompleteList}
+						autocompleteList={config.autocompleteList?.filter(
```

Comment:
should we change how this variable is called in the config? 
Because this line has the real "autocompleteList". Before it we just pass down the sessionsStore.booksStream, and use it for slightly different purposes (like validation in Bubble components), other than autocompletion

Reply:
Just rename property of AutocompleteInput component "autoCompleteList" into "autocompleteList"